lib/cra-template/template.json

```diff
@@ -0,0 +1,18 @@
+{
+  "package": {
+    "scripts": {
+      "storybook": "start-storybook -p 6006",
+      "build-storybook": "build-storybook"
+    },
+    "dependencies": {
+      "@testing-library/react": "^9.3.2",
+      "@testing-library/jest-dom": "^4.2.4",
+      "@testing-library/user-event": "^7.1.2",
+      "@storybook/addon-actions": "^6.0.0-alpha.25",
+      "@storybook/addon-docs": "^6.0.0-alpha.25",
+      "@storybook/addons": "^6.0.0-alpha.25",
+      "@storybook/preset-create-react-app": "^6.0.0-alpha.25",
```

Comment:
Sorry, this is actually wrong - I missed it before. `2.1.1` is the latest.

Reply:
Ahh, my bad! Will fix it soon